examples/codesandbox-starter/src/theme.js

```diff
@@ -0,0 +1,38 @@
+/** @jsx jsx */
+
+import { Global } from '@emotion/core'
+import React, { memo } from 'react'
+import { jsx, useThemeUI, ThemeProvider as TP, Styled } from 'theme-ui'
+import { base } from '@theme-ui/presets'
+
+const ThemeProvider = memo(({ children, ...props }) => (
+  <TP theme={base} {...props}>
+    <Styled.root>{children}</Styled.root>
+  </TP>
+))
+
+const Reset = () =>
+  React.createElement(Global, {
+    styles: {
+      body: {
+        margin: '0',
```

Comment:
It'd be great if the only global style applied here was for the body margin, since the other styles can be handled with Emotion